tests/src/test/java/org/neo4j/graphalgo/impl/metaPathComputationTests/GetSchemaTest.java

```diff
@@ -0,0 +1,130 @@
+package org.neo4j.graphalgo.impl.metaPathComputationTests;
+
+import org.junit.*;
+import org.neo4j.graphalgo.TestDatabaseCreator;
+import org.neo4j.graphalgo.core.GraphLoader;
+import org.neo4j.graphalgo.core.heavyweight.HeavyGraph;
+import org.neo4j.graphalgo.core.heavyweight.HeavyGraphFactory;
+import org.neo4j.graphalgo.impl.metaPathComputation.getSchema.GetSchema;
+import org.neo4j.graphalgo.metaPathComputationProcs.GettingStartedProc;
```

Comment:
"GettingStartedProc"?

Reply:
Great Question. I don't now why this works, but when I tried changing it things broke. We use GettingStartedProc in pretty much all our Test-Files